hack/README.md

```diff
@@ -0,0 +1,70 @@
+docker/hack
+============================
+
+## About
+
+`hack` contains a collection of scripts used to build and manage this
```

Comment:
I'd change `hack` to `This directory`, or "The `hack` directory contains ..." Similar to what we do for the contrib directory